src/hotspot/os_cpu/linux_zero/os_linux_zero.cpp

```diff
@@ -24,7 +24,7 @@
  */
 
 #include "asm/assembler.inline.hpp"
-#include "atomic_linux_zero.hpp"
+#include "atomicAccess_linux_zero.hpp"
```

Comment:
The platform-specific atomic[Access] headers were never really intended for direct inclusion.
I think it's being included here to get atomic_copy64. But there is code in this file that I think
*should* be using `AtomicAccess`. But that's a pre-existing issue that doesn't need to be dealt
with in this PR. But I'd prefer this included `atomicAccess.hpp`.

Reply:
I've updated the file to use atomicAccess.hpp, but I'll let someone else make the move of the function if that is deemed to be the right thing to do.

Reply:
https://bugs.openjdk.org/browse/JDK-8367717 "Cleanup atomic_copy64"